component/agent.jsonnet

```diff
@@ -165,6 +187,23 @@ local metricsService = loadManifest('manager/service.yaml') {
       },
     ],
   },
+  '01_default_org_role_binding': [
+    kube.ClusterRoleBinding('appuio-cloud-agent:%s' % [ cr ]) {
```

Comment:
Would be nice to add a comment here why this is necessary (the agent needs all permissions it wants to grant in new RoleBindings).